src/main/webapp/comment-tests.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
General comment on UI and branding. No comment on whether you should change what you have now or not.

https://www.youtube.com/about/brand-resources/
https://developers.google.com/youtube/terms/branding-guidelines